src/main.c

```diff
@@ -666,6 +668,9 @@ poe_port_setup(void)
 {
 	size_t i;
 
+	poe_cmd_port_disconnect_type(PORT_ID_ALL, 2);
+	poe_cmd_port_detection_type(PORT_ID_ALL, 3);
```

Comment:
The documentation is less certain on this point about the _connect_ detection type setting, but I lack deep knowledge of the 802.3a{f,t} protocol to say, so this is fine.

Reply:
That's exactly why I'm not changing the setting itself. Just how we send it to the MCU...